### DISCUSSION TITLE
Configure application name without changing crate name?

### DISCUSSION THREAD

**Discussion by Bendrien** [slint-ui/slint#561] [Q&A]
Sixtyfps apps in os process overview are currently listed after their crate name. Is there a way to change the shown name without changing the crate name?

**Comment by tronical** [comment]
Which operating system are you using?

**Reply by Bendrien** [reply]
Windows

**Comment by ogoffart** [comment]
I think you can change the binary name in the `[[bin]]` section of `cargo.toml`
https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries

Or are you talking about something else?
Either way, i think there might be some API to change that, but i don't think it is related to SixtyFPS itself.

**Reply by Bendrien** [reply]
This seem to work, thanks for the hint! But there are some caveats:

* Some characters seem to be disallowed. "Foo Bar" for example does not work due to the space in between.
* Using for example "Foo-Bar" you get a warning:  `crate "Foo-Bar" should have a snake case name`...

Easiest would be just to configure the application name via the sixtyfps `Window.title` property or maybe even a dedicated one. Does this make sense?
